webapp/channels/src/components/user_groups_modal/user_groups_modal.scss

```diff
@@ -388,12 +388,18 @@
                 padding: 12px 32px 24px;
             }
 
-            h2 {
-                padding: 0 32px;
-                color: inherit;
-                font-size: 18px;
-                font-weight: 600;
-                line-height: 24px;
+            .group-add-people-fieldset {
+                margin-top: 20px;
+
+                legend {
```

Comment:
The gap between this text and the input are a bit larger on the branch than master